.pyenchant_dessia.dict

```diff
@@ -53,3 +53,6 @@ uuid
 uuids
 visjs
 volmdlr
+Dessia's
```

Comment:
This document should be kept alphabetically ordered

Reply:
it's done